.github/workflows/release.yml

```diff
@@ -1,71 +1,91 @@
-name: Release
+name: release-please
 
 on:
-  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
This action runs on all commits on master. `release-please` looks into if it requires opening a PR or publishing (in the case of PR merge commit)